src/store/initialState.js

```diff
@@ -3,13 +3,21 @@ const initialState = {
     user: {},
     signin: {
       loading: false,
-      error: ''
+      error: ""
     },
     signup: {
       loading: false,
       error: ''
     },
     isAuthenticated: false
+  },
+  article: {
+    article: {
+      author: {},
+      tags: [{}]
+    },
+    isLoading: true,
```

Comment:
I think the default state for `isLoading` should be set to false, since you dispatch an action to set this when you start fetching the article. Also you can rename this state to `loading` in order to maintain consistency with the rest of the codebase.

Reply:
Thanks @CEOehis I'll fix this.